src/routes/add-place.ts

```diff
@@ -31,6 +31,19 @@ export default async function addPlace(fastify: FastifyInstance) {
     return resp.view('src/liquid/app/view.html', tmplData);
   });
 
+  fastify.post('/widget', async (req, resp) => {
```

Comment:
is widget a descriptive name for what this api endpoint is doing? i don't think we use that language anywhere else ... what is a widget?

this is for dob only, right? i think the name should likely include dob..
don't all routes in this file start with `/place`? should this also?

Reply:
> is widget a descriptive name for what this api endpoint is doing? i don't think we use that language anywhere else ... what is a widget?

oh I just left it in as i was experimenting with how we'd support this kind of inputs in the future. I'm not really sure where this endpoint should live but I've renamed it to `/place/dob`.

Reply:
it is only used by the app-place form, so having it in here seems "reasonble"
the routes aren't super clearly designated. if you ever want to clean it up i welcome it!